demo/src/App.js

```diff
@@ -46,6 +62,15 @@ export default class App extends PureComponent {
     });
   }
 
+  toggleChannel(channel) {
```

Comment:
```suggestion
  toggleChannel(channelName) {
```
... if that's actually what it is?